aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Cloud authentication with Azure Active Directory B2C
+author: camsoper
+description: Shows how to set up Azure Active Directory B2C with ASP.NET Core
+keywords: ASP.NET Core,authentication,AAD,B2C,Azure Active Directory,Active Directory
+ms.author: casoper
+manager: wpickett
+ms.date: 01/12/2018
+ms.topic: article
```

Comment:
Is this an article or a tutorial?

Reply:
What's the difference?  Back in Azure docs world, everything was either article or reference.

Reply:
If it's a walkthrough or tutorial, you'll need the MVC "During the course of this tutorial, you learn:" checklist. Example here: https://docs.microsoft.com/en-us/dotnet/core/docker/docker-basics-dotnet-core